### ISSUE TITLE
Delete repository

### ISSUE THREAD

**Issue by Aniket21mathur** [sugarlabs/flappy-birds-activity#16]
The following changes can be made:

- The vertical gap between the top and botton pole and the horizontal distance between two consecutive sets can be increased.
- The speed of the bird can be reduced when the game starts and can be increased as the game progress.
- The bird control can be shifted from upper arrow key to space bar.
I suggest these changes to make the game bit easier so that the user is kept motivated.I myself played 10 times and my best was 4 :sweat_smile: .It might be possible that I am a really bad player :sweat_smile: :sweat_smile:

**Comment by Aniket21mathur** [comment]
@quozl What are your opinions in it. :sweat_smile:

**Comment by quozl** [comment]
> The vertical gap between the top and botton (sic) pole and the horizontal distance between two consecutive sets can be increased.

Under what conditions?

> The speed of the bird can be reduced when the game starts and can be increased as the game progress.

Yes.

> The bird control can be shifted from upper (sic) arrow key to space bar.

Either key.

**Comment by Aniket21mathur** [comment]
@quozl the vertical gap and horizontal gap can be increased for all conditions to make the game a bit easier to play for the users :sweat_smile:.

**Comment by quozl** [comment]
I see, thanks.

**Comment by Aniket21mathur** [comment]
@quozl finally what changes we have to make? Thanks!

**Comment by quozl** [comment]
I understand the issue you've raised.  The game is difficult to play and requires mastery.  Your suggestions will lower initial difficulty but yet still enable mastery.  Will you provide controls for difficulty as well?  Where will you store the number of games played; in the activity instance data directory, or in the journal entry?  Advantage to the latter is deleting the journal entry restarts from initial difficulty.

**Comment by Aniket21mathur** [comment]
@quozl  I am sorry I am not able to clearly undersatnd 
"Will you provide controls for difficulty as well?"
Is there any need to store the no of games played? Please clarify.
Thanks!

**Comment by quozl** [comment]
Sure.

> "Will you provide controls for difficulty as well?"

What you proposed earlier in the issue is that the game start easy and become harder over time.  This means there is some internal state which you might call _difficulty_.  Showing this state on display may be helpful to a player.  Allowing the state to be changed by the player may also be helpful.  A control is a UI widget.

> Is there any need to store the no of games played?

When the activity is stopped, and then started again, the internal state which you might call _difficulty_ will either reset to the default, or reset to the previous value.  One way to reset to a previous value is to store the number of games

You've proposed a feature, and we're working toward how you might implement the feature.

**Comment by nswarup14** [comment]
May I suggest having three different levels of difficulty for the user to choose from? When the user feels he has a certain amount mastery over a particular level, he can switch to a higher difficulty level. 
As @Aniket21mathur  pointed out, each level can vary in terms of
1) Speed of the bird
2) Distance between two sets of pillars
3) Gap between the upper and lower pillar

This might prove a better experience to the user, as I have noticed that several mobile apps have a similar kind of features.
I welcome further discussion on this topic.

**Comment by Aniket21mathur** [comment]
Thanx @nswarup14 for your suggestion, it will be good to break the game into three levels of mastery, 
varying the mentioned parameters, any suggestions for other  parameters that should be changed between levels.
@quozl  What you have to say in this?

>  Where will you store the number of games played; in the activity instance data directory, or in the journal entry? 

I think that it would be better to save the progress made by user in the activity instance data directory

**Comment by quozl** [comment]
I've nothing further to say; except; please continue to work toward consensus.

**Comment by rhl-bthr** [comment]
This activity is similar to https://github.com/sugarlabs/flappy.

I have implemented different levels in https://github.com/sugarlabs/flappy/commit/c1e8a11d56ccfe37a9cfb8b6ab5bff5f70930ea4

Here are my suggestions,
 - [ ] see what's different between the two, (also review git histories)
 - [ ] implement the features missing in one but present in the other
 - [ ] delete the other repository

**Comment by Aniket21mathur** [comment]
Thanks @pro-panda for your suggestions.I will soon start working on it!

**Comment by quozl** [comment]
I agree with @pro-panda.  I had not noticed the duplication until now.

**Comment by iamutkarshtiwari** [comment]
I don't think the aforementioned repositories are similar in any way but the concept.
The graphics and gameplay of this activity are almost the same as the actual flappy bird android app. 
The time I wrote this activity, I did a terrible job hardcoding dimensions which makes it difficult to scale/play this on different sized displays. Apologies for that.

Thanks to @AlanJAS for helping me port some of my first Sugar activities.
